test/http-delivery-mocha.js

```diff
@@ -111,4 +112,49 @@ describe('http-delivery', function() {
             });
         }).should.eventually.have.property('data').that.has.property('@id', "http://dbpedia.org/resource/John_Lennon");
     });
+    xit("should round trip rdf file through SPARQL LOAD", function() {
```

Comment:
Is this actually round tripping Turtle?  I am not seeing where the data is read back from the SPARQL store.  It looks like it is checking for JSON results on line 158, rather than Turtle.